Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivityModule.kt

```diff
@@ -6,15 +6,15 @@ import de.rki.coronawarnapp.ui.interoperability.InteroperabilityConfigurationFra
 import de.rki.coronawarnapp.ui.interoperability.InteroperabilityConfigurationFragmentModule
 import de.rki.coronawarnapp.ui.main.home.HomeFragmentModule
 import de.rki.coronawarnapp.ui.onboarding.OnboardingDeltaInteroperabilityModule
-import de.rki.coronawarnapp.ui.submission.fragment.SubmissionQRCodeInfoFragment
-import de.rki.coronawarnapp.ui.submission.fragment.SubmissionQRCodeInfoModule
 import de.rki.coronawarnapp.ui.riskdetails.RiskDetailsFragmentModule
+import de.rki.coronawarnapp.ui.submission.SubmissionActivityModule
 
 @Module(
     includes = [
         OnboardingDeltaInteroperabilityModule::class,
         HomeFragmentModule::class,
-        RiskDetailsFragmentModule::class
+        RiskDetailsFragmentModule::class,
+        SubmissionActivityModule::class
```

Comment:
Small nitpick, let's call it `SubmissionFragment*s*Module`, as has nothing to do with activities.

Reply:
Of course! I'll change that now